package.json

```diff
@@ -15,10 +15,10 @@
         "underscore": "1.x"
     },
     "peerDependencies": {
-        "@havenlife/supertype": "3.3.*"
+        "@havenlife/supertype": "3.x"
     },
     "devDependencies": {
-        "@havenlife/supertype": "3.3.*",
+        "@havenlife/supertype": "3.x",
```

Comment:
while I agree we can make this change, it is a pretty significant change in behavior so maybe add this to description in history.md?

Reply:
Yeah fair enough